coverage

```diff
@@ -3,13 +3,12 @@
 echo "----Running Code Coverage----"
 
 # Remove multiprocessing coverage files in case a previous combine wasn't performed
-rm .coverage.*
 rm -fR cover/
 # Remove the main coverage file (.coverage)
 coverage erase
 
 # Discover and run all tests
-coverage run --concurrency=multiprocessing ./scripts/run_tests.py discover -v
```

Comment:
Can we then remove this file? `/scripts/run_tests.py`

Reply:
Yes, removed now.